src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -21,6 +22,19 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     logger.infoRequest(req, `${config.BENEFICIAL_OWNER_TYPE_PAGE} requiresTrusts=${requiresTrusts}`);
 
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL)) {
+      return res.render(config.BENEFICIAL_OWNER_TYPE_PAGE, {
+        FEATURE_FLAG_ENABLE_REDIS_REMOVAL: true,
```

Comment:
I think we should add some comments in around this, to explain why it's not being done in the usual way (to facilitate testing). I can do that now.

Reply:
Done.